custom_components/octopus_intelligent/graphql_client.py

```diff
@@ -191,7 +191,7 @@ async def __async_cancel_boost_charge(self, session, account_id: str):
     result = await session.execute(query, variable_values=params, operation_name="deleteBoostCharge")
     return result['deleteBoostCharge']
 
-  async def __async_get_accounts(session):
+  async def __async_get_accounts(self, session):
```

Comment:
See explanation in a [PR 38 comment](https://github.com/megakid/ha_octopus_intelligent/pull/38/files#r1510344287) where this same fix was also required.